sky/engine/core/text/ParagraphBuilder.cpp

```diff
@@ -136,7 +136,7 @@ ParagraphBuilder::~ParagraphBuilder()
 {
 }
 
-void ParagraphBuilder::pushStyle(Int32List& encoded, const String& fontFamily, double fontSize, double letterSpacing, double wordSpacing, double lineHeight)
+void ParagraphBuilder::pushStyle(Int32List& encoded, const char* fontFamily, double fontSize, double letterSpacing, double wordSpacing, double lineHeight)
```

Comment:
I meant a U+0000, as opposed to a null pointer.

We need to handle nulls and out-of-range values in general in our APIs but I figure we'll deal with that all at once at some point.